docs/user/adding-glean-to-your-project.md

```diff
@@ -11,16 +11,26 @@ Products using the Glean SDK to collect telemetry **must**:
 
 ## Usage
 
-### Setting up the dependency
+### Integrating with your project
+
+There are currently two implementations of Glean: an Android-only implementation that is part of Mozilla's [android-components](https://github.com/mozilla-mobile/android-components) project, and a [cross-platform implementation in Rust](https://github.com/mozilla/glean).
```

Comment:
I would remove this part. We can add references to the Rust implementation once we have a clearer idea of how to integrate: we already know that all the Mozilla apps would keep consuming Glean off android components, regardless of the implementation. I'd limit the changes to this PR to fixing the links and the repos.

Reply:
Sure, I mostly agree, but there was confusion from a user about whether this document applies to a-c (since it exists in a different repo).  Can probably do without, though, as you say.